.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: node_js
-cache: yarn
+cache:
+  directories:
+    - node_modules
 notifications:
   email: false
 node_js:
   - "4"
   - "6"
   - "8"
+before_install:
+  - if [[ `npm -v` != 5* ]]; then npm install -g npm@5; fi
 script:
   - npm test
```

Comment:
We could even remove the script step since `npm test` is the default one. But it's nitpicking. :)

Reply:
Well while we're at it :)